Dockerfile

```diff
@@ -0,0 +1,96 @@
+# Base container that is used for both building and running the app
+FROM centos:latest as base
+ENV RUBY_SCL="rh-ruby25"
+ENV NODEJS_SCL="rh-nodejs8"
+
+RUN \
+  echo "tsflags=nodocs" >> /etc/yum.conf && \
+  yum -y upgrade && \
+  yum -y install centos-release-scl epel-release && \
+  yum -y install ${RUBY_SCL} ${NODEJS_SCL}-nodejs \
+     mariadb-libs postgresql-libs \
+    ${RUBY_SCL}-ruby{,gems} ${RUBY_SCL}-rubygem-{rdoc,rake,bundler} nc && \
+  yum clean all && \
+  rm -rf /var/cache/yum/
+
+ENV HOME=/home/foreman
+WORKDIR $HOME
+RUN groupadd -r foreman -f -g 1001 && \
+    useradd -u 1001 -r -g foreman -d $HOME -s /sbin/nologin \
+    -c "Foreman Application User" foreman && \
+    chown -R 1001:1001 $HOME
+
+# Add a script to be executed every time the container starts.
+COPY entrypoint.sh /usr/bin/
```

Comment:
We could get rid of this if we don't use the SCL. That's a huge gain IMHO because it simplifies the usage.

Reply:
TBH that's not just for SCL, i think its useful to ensure a certain script runs before other commands...